workspaces/vanilla/porter.yaml

```diff
@@ -22,7 +22,7 @@ parameters:
     description: "Unique 4-character long, alphanumeric workspace ID"
     minLength: 4
     maxLength: 4
-  - name: location
+  - name: azure_location
```

Comment:
This change is missing from `.env.sample` and `/docs/authoring-workspaces.md`

Reply:
fixed

Reply:
`/docs/authoring-workspaces.md` not updated?

Reply:
That one was already change by Tess yesterday, so it's already merged develop.